### ISSUE TITLE
Brotlin for compression

### ISSUE THREAD

**Issue by Serkan-devel** [AppImage/AppImageSpec#14]
As I understand how Appimages work, is that they are compressed archives which have an executable component. Google has created a new compression algorithm called Brotlin,  which is open source and compresses even better than gzip. Using that compression algorithm on their play store saved then around 1.5 Petabytes in bandwith, it might also help compression of appimages. While compressing might take a little bit longer than on gzip, decompressing shouldn't take that long, which shrinks the appimage file size. Could this work?

**Comment by probonopd** [comment]
Yes, this could work, if someone implements Brotli compression for squashfs (or another filesystem image format). Looks like someone already had this idea: https://www.topcoder.com/challenges/30053205/?type=develop but I could not find the code.

**Comment by Serkan-devel** [comment]
Any updates on this?

**Comment by probonopd** [comment]
It is currently not a priority for us. Is it for you?

**Comment by Serkan-devel** [comment]
idk
